include/sound/sof/xman.h

```diff
@@ -39,8 +39,8 @@ struct xman_header {
 
 /* Extended manifest elements identificators */
 enum xman_elem_type {
-	FOO
-};
+	XMAN_ELEM_CC_VERSION	= 3,
```

Comment:
fix commit message
"As it is build-time known information is should be pass throu externded manifests instead of mailbox."

->

"The compiler version and description can be extracted from the extended manifest content. This information known at build-time does not need to be provided in a mailbox"

Reply:
changed